RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -750,7 +762,10 @@ int handlePulls(int day)
 			{
 				pullXWhenHaveY($item[snow suit], 1, 0);
 			}
-			if(!possessEquipment($item[Snow Suit]) && !possessEquipment($item[Filthy Child Leash]) && !possessEquipment($item[Astral Pet Sweater]) && glover_usable($item[Filthy Child Leash]))
+			boolean famStatEq = possessEquipment($item[fuzzy polar bear ears]) || possessEquipment($item[miniature goose mask]) || possessEquipment($item[tiny glowing red nose]);
```

Comment:
Currently the famStatEq isn't pulled. Please add logic to pull the best equip instead of only checking if we already have it.

Reply:
logic for this was assuming if user pulled one of the stat ones then that's the one they want but I don't know if it's the best, filthy child leash does damage instead which can be very helpful

Reply:
We should add logic to determine which is best. Goal of autoscend is to be as hands free as possible.

Can you elaborate on where stench damage is useful. NS tower?

Reply:
it's useful in combat, can be the difference between if the script manages to get through friars or giants on its own or not, so it's hard to judge which is more needed but damage is probably the safer auto pull